x/foundation/msgs_test.go

```diff
@@ -127,15 +124,13 @@ func TestMsgUpdateMembers(t *testing.T) {
 			valid: true,
 		},
 		"empty operator": {
-			operator: sdk.AccAddress{},
 			members: []foundation.Member{{
 				Address:       addrs[1].String(),
 				Participating: true,
 			}},
 		},
 		"empty members": {
 			operator: addrs[0],
-			members:  []foundation.Member{},
```

Comment:
https://github.com/line/lbm-sdk/blob/1bc3aac987c712a4ba9c980625980c3b8d4982ab/x/foundation/msgs_test.go#L134

Reply:
These changes were added by accident. As you stated, the purpose of this PR is minimizing the differences. So I should apply these changes in another PR.